credential-token_polygon/README.md

```diff
@@ -0,0 +1,27 @@
+# The WOMEN IN WEB3 Member Portal
+A token-gated website for WIW3 members to go through the Venture Builder and earn credential tokens as they complete specific milestones.
+
+## Getting Started
+
+### Part 1: Alchemy, Crypto Wallet & Install
```

Comment:
Is there a reason why we need two different README files? Wondering if we can condense it into one?

Reply:
I guess we don't really need it, I added two different READMEs for each directory level of the project